src/qmapshack/mouse/CMouseSelect.cpp

```diff
@@ -168,11 +169,11 @@ void CMouseSelect::draw(QPainter& p, CCanvas::redraw_e needsRedraw, const QRect
         item->drawHighlight(p);
     }
 
-    for(const poi_t& poiItem : poisFound)
+    for(const QPointF& pos : posPOIHighlight)
```

Comment:
Do it as the other loop. As pos is copied anyway to pxPos and not needed anywhere else it's ok to do the copy in the iteration.